_includes/opengraph.html

```diff
@@ -5,3 +5,5 @@
   <meta property="og:image" content="{% if page.social_image %}{{ page.social_image }}{% else %}https://images.digital-democracy.org{{ page.image | replace: '.jpg', '-1200.jpg' }}{% endif %}" />
   <meta property="og:image:secure_url" content="{% if page.social_image %}{{ page.social_image }}{% else %}https://images.digital-democracy.org{{ page.image | replace: '.jpg', '-1200.jpg' }}{% endif %}" />
   <meta property="fb:app_id"  content="808970299184874" />
+  <meta property="twitter:card"  content="summary" />
```

Comment:
We should make this content=""{{ page.excerpt | strip_html }}" just to be consistent

Reply:
This is a twitter-specific tag that takes one of the following values: summary, summary_large_image, app, player ([reference](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/abouts-cards))

Reply:
On that note, we might want to use `summary_large_image` if the Dd logo is cropped into a square: https://stackoverflow.com/questions/47578130/twitter-card-summary-vs-summary-large-image

Reply:
> This is a twitter-specific tag that takes one of the following values: summary, summary_large_image, app, player ([reference](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/abouts-cards))

Got it!

Reply:
> On that note, we might want to use `summary_large_image` if the Dd logo is cropped into a square: https://stackoverflow.com/questions/47578130/twitter-card-summary-vs-summary-large-image

Yeah! Lets add this when we get access to the images from Gregor!